src/java/org/apache/cassandra/tools/nodetool/CompactionStats.java

```diff
@@ -53,29 +58,82 @@ public class CompactionStats extends NodeToolCmd
     public void execute(NodeProbe probe)
     {
         PrintStream out = probe.output().out;
+        out.print(pendingTasksAndConcurrentCompactorsStats(probe));
+        out.print(compactionsCompletedStats(probe));
+        out.print(compactionThroughPutStats(probe));
+        out.println();
         CompactionManagerMBean cm = probe.getCompactionManagerProxy();
+        reportCompactionTable(cm.getCompactions(), probe.getCompactionThroughputBytes(), humanReadable, vtableOutput, out);
+    }
+
+    private static String pendingTasksAndConcurrentCompactorsStats(NodeProbe probe)
+    {
         Map<String, Map<String, Integer>> pendingTaskNumberByTable =
             (Map<String, Map<String, Integer>>) probe.getCompactionMetric("PendingTasksByTableName");
-        int numTotalPendingTask = 0;
+        StringBuffer toPrint = new StringBuffer();
+        toPrint.append(String.format("%s concurrent compactors, %s pending tasks", probe.getConcurrentCompactors()
+        , numPendingTasks(pendingTaskNumberByTable)));
```

Comment:
I think there is a problem of code format
line 75 should have some space before ',' , so that In this way “，”  can be aligned with "%$"

Reply:
yeah just align it like this

    toPrint.append(String.format("%s concurrent compactors, %s pending tasks",
                                 probe.getConcurrentCompactors(), 
                                 numPendingTasks(pendingTaskNumberByTable)));